frontend/js/wordInteractions.js

```diff
@@ -135,6 +136,7 @@ function playerTypesWord () {
     if (removeWordFromPage(typedSubmission)) {
       game.wordsTyped.push(typedSubmission);
       game.score += typedSubmission.length;
+      document.getElementById('score').textContent = game.score;
```

Comment:
nice. In line 142 below we also made the score decrease if someone entered a typo, so you probably want to repeat this line there or move it out of the if/else statement

Reply:
Good catch, fixed that!